apps/vetrina/.env.development

```diff
@@ -1,2 +1,3 @@
 PERSONA_URL=https://persona.staging.ksfmedia.fi/v1
 BOTTEGA_URL=https://bottega.staging.ksfmedia.fi/v1
+SENTRY_DSN=
```

Comment:
I'm guessing this value depends on the local dev environment configuration?

Reply:
Yeah. Let's not commit the value here